lbrynet/core/client/DownloadManager.py

```diff
@@ -59,8 +58,8 @@ def check_stop(result, manager):
         d1.addBoth(check_stop, "progress manager")
         d2 = self.connection_manager.stop()
         d2.addBoth(check_stop, "connection manager")
-        dl = defer.DeferredList([d1, d2])
-        dl.addCallback(lambda xs: False not in xs)
+        dl = defer.DeferredList([d1, d2], fireOnOneErrback=True, consumeErrors=True)
```

Comment:
Reconsidering - `fireOnOneErrback` should probably be False so that the other deferred (the one that didn't fail) can continue doing its thing. I'll fix that.

`consumeErrors` should be set, otherwise if, for example, `d1` fails the error will be unhandled by d1 (since there are no errbacks attached to d1) and will also get passed along to possible errbacks of `dl`.  `consumeErrors` helps prevent spurious 'Unhandled error in Deferred' messages.